### DISCUSSION TITLE
AP or mixed mode implementation

### DISCUSSION THREAD

**Discussion by SamGeens** [JiriBilek/ESP_ATMod#3] [General]
Hi 

Are there any plans to ever implement AP or mixed mode?

**Comment by JiriBilek** [comment]
For now, I am not planning to extend the command set to AP modes. I have no application to use and test it with.
Sorry.

**Reply by SamGeens** [reply]
I've a use case where it could come handy to have a more userfriendly way to setup up or change a wifi connection. I planned to have my arduino start in AP mode in which I could give in the SSID and password on a webpage. 

If you're just not implementing it this is fine but if the hurdle is just the testing part I could help you with that.

**Comment by JiriBilek** [comment]
I understand. I have more important things to do now so I can return to this project in a month, I hope. I you will be still interested, I'd like your help with testing.

**Reply by SamGeens** [reply]
Great! I do completely understand that there are more important things to do. As said before, for me it is just a convenience issue that I hopefully will not have to set up wifi by code over and over again. But I can imagine that this could be a step to many more use cases. 

You can find me in this discussion when you do have the time for it and I'll be happy to help.
